components/org.wso2.carbon.identity.application.authenticator.cognito/pom.xml

```diff
@@ -0,0 +1,123 @@
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.eclipse.osgi</groupId>
+            <artifactId>org.eclipse.osgi.services</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.logging</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.framework</groupId>
+            <artifactId>org.wso2.carbon.identity.application.authentication.framework</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.framework</groupId>
+            <artifactId>org.wso2.carbon.identity.application.common</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.outbound.auth.oidc</groupId>
+            <artifactId>org.wso2.carbon.identity.application.authenticator.oidc</artifactId>
+        </dependency>
+    </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-scr-plugin</artifactId>
+                <version>${maven.scr.plugin.version}</version>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>${maven.bundle.plugin.version}</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Private-Package>
+                            org.wso2.carbon.identity.application.authenticator.cognito.internal
+                        </Private-Package>
+                        <Import-Package>
+                            javax.servlet.http; version="${imp.pkg.version.javax.servlet}",
+
+                            org.apache.oltu.oauth2.client; version="${oltu.package.import.version.range}",
+                            org.apache.oltu.oauth2.client.request; version="${oltu.package.import.version.range}",
+                            org.apache.oltu.oauth2.client.response; version="${oltu.package.import.version.range}",
+                            org.apache.oltu.oauth2.common.exception; version="${oltu.package.import.version.range}",
+                            org.apache.oltu.oauth2.common.message.types;
+                            version="${oltu.package.import.version.range}",
+                            org.apache.oltu.oauth2.common.utils; version="${oltu.package.import.version.range}",
+                            org.wso2.carbon.utils.*; version="${carbon.kernel.package.import.version.range}",
+                            net.minidev.json; version="${net.minidev.json.imp.pkg.version.range}",
```

Comment:
is minidev.json needed here ?

Reply:
It is used in OpenIDConnectAuthenticator https://github.com/wso2-extensions/identity-outbound-auth-oidc/blob/master/components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java#L20

Reply:
Which means we do not need it here